python/osrd_schemas/osrd_schemas/infra.py

```diff
@@ -433,6 +443,27 @@ class Panel(TrackLocationTrait):
     value: Optional[NonBlankStr] = Field(description="If the panel is an announcement, precise the value(s)")
 
 
+class DeadSection(BaseObjectTrait):
+    """
+    Dead zones (or sections) are sections of track where electrical trains cannot pull power from catenaries.
+    Instead, trains have to rely on inertia to cross such sections.
+
+    If the section is designated as a pantograph drop zone, trains will lower pantographs before entering the section,
+    and will start raising pantographs again once the train has entirely left the section.
+
+    If a train has a pantograph which is not located at its head (we talk of push-pull trains) and moving in reverse
+    mode, it will have to wait a special "REV" panel (for reversible) to start pulling power from catenaries again.
+    """
```

Comment:
Nice documentation ! I feel like it might be a little too specific to the SNCF use case, but it's very explicit so it's a nice doc

Reply:
Victor helped me to write it :star_struck: